financial.org

```diff
@@ -13,17 +13,17 @@ Wait a second! Stop right there, loan officer! I have calc!
 
 #+BEGIN_EXAMPLE
 M-x calc
-500000  (the amount of the loan)
-30   (the number of payments)
 '5%  (equivalent to typing 0.05)
+30   (the number of payments)
+500000  (the amount of the loan)
 b M  (calc-fin-pmt, computing the amount of periodic payments to amortize a loan)
 
 Result:
-1:  25,000
+1: 14,2066908
```

Comment:
Your ordering looks correct, but I get $32,525.72!
Can you try this again?

Reply:
I'm fairly sure your number can't be right, because $14,206 * 30 = $426,000, which is less than the amount of the loan.  The total amount of payments has to be greater, right?

However, what calc gives me seems pretty high - a total value of $975,771.

Reply:
You're right, sorry about that. The correct amount is $32,525.72. 

What's a bit confusing is the notion of time. Calc expects the interest rate and the number of payments to refer to the same temporal units (months or year, but you have to be consistent).
So your exemple is either a monthly payment with 5% interest each month or a yearly payment with 5% interest each year. Either case, it's expensive because you have to pay 5% each month (or year). So the amount does not surprise me.

However, I've spent too much time getting confused with the monthly or yearly interest rate so I've taken the liberty to add another example in the new commit.